src/components/userProfile/UserProfile.tsx

```diff
@@ -6,11 +6,14 @@ import {
   Typography,
   Button,
 } from '@mui/material'
-import CircleRoundedIcon from '@mui/icons-material/CircleRounded'
+
 import { makeStyles } from 'tss-react/mui'
+import CircleRoundedIcon from '@mui/icons-material/CircleRounded'
+import FotoCarousel from './FotoCarousel'
 
 const Card = () => {
```

Comment:
plz, change name to `UserProfile`. I renamed couple of files bc naming were very generic

Reply:
Done